base/applications/network/dwnl/lang/tr-TR.rc

```diff
@@ -0,0 +1,29 @@
+/* Translator:
+2025 Efe Gürel (efegurel) (info [dot] efegurel [at] gmail [dot] com) 
+*/
```

Comment:
```suggestion
/*
 * PROJECT:     ReactOS File Download Utility
 * LICENSE:     See COPYING in the top level directory
 * PURPOSE:     Turkish resource file
 * TRANSLATOR:  Copyright 2025 Efe Gürel <info.efegurel@gmail.com>
 */
```

Reply:
You have force-pushed, and (unintentionally?) reverted previously applied review suggestions.